src/object_classes_permissions.md

```diff
@@ -974,9 +975,15 @@ type_transition sysadm_t sysadm_t : anon_inode uffd_t "[userfaultfd]";
 allow sysadm_t uffd_t:anon_inode { create };
 ```
 
-Currently only ***userfaultfd**(2)* makes use of this service (from kernel 5.12)
-as described in the patch series
-<https://lore.kernel.org/selinux/20210108222223.952458-1-lokeshgidra@google.com/>
+The current implementations that make use of this service are:
```

Comment:
Linux v6.0 also uses the LSM/SELinux anonymous inode controls for [`memfd_secret()`](https://lwn.net/Articles/865256).

* https://www.paul-moore.com/blog/d/2022/10/linux_v60.html